examples/volumes/nfs/README.md

```diff
@@ -85,7 +85,8 @@ Replace the invalid IP in the [nfs PV](nfs-pv.yaml). (In the future,
 we'll be able to tie these together using the service names, but for
 now, you have to hardcode the IP.)
 
-Create the the [persistent volume](https://kubernetes.io/docs/user-guide/persistent-volumes.md)
+Create the [persistent volume](https://kubernetes.io/docs/user-guide/persistent-volumes.md)
```

Comment:
Seems we move it to  https://kubernetes.io/docs/concepts/storage/persistent-volumes/ ?